README.md

```diff
@@ -6,6 +6,26 @@
 
 The Python [OpenTelemetry](https://opentelemetry.io/) client.
 
+## Installation
+
+## Usage
+```python
+from opentelemetry import trace
+from opentelemetry.context import Context
+from opentelemetry.sdk.trace import Tracer
+
+trace.set_preferred_tracer_implementation(lambda T: Tracer())
+tracer = trace.tracer()
+with tracer.start_span('foo'):
```

Comment:
what is the expected behavior when one starts a span with the same name? it looks like it'll override the current span, which means that it will proceed to replace the contents of the outer span of the same name.

side question: are named spans valuable?

Reply:
> what is the expected behavior when one starts a span with the same name? it looks like it'll override the current span, which means that it will proceed to replace the contents of the outer span of the same name.

It will just work. What we put in the context is the `span` instead of the `name`. `name` doesn't matter.

> side question: are named spans valuable?

Definitely YES! Span name is actually the operation name (or the name you see from the Gantt Chart).